### PR TITLE
Characterization 139

### PR DESCRIPTION
**Pull request by grosscol**
Will squash commits prior to merge.

This PR is pending projecthydra-labs/hydra-works#209

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ gem 'slop', '~> 3.6.0' # This just helps us generate a valid Gemfile.lock when R
 
 gem 'curation_concerns-models', path: './curation_concerns-models'
 
+gem 'hydra-works', git: 'https://github.com/grosscol/hydra-works.git', branch: 'characterization'
```
Comment (jcoyne) [inline Gemfile]: Can you switch this to something in projecthydra?

File: curation_concerns-models/app/jobs/characterize_job.rb [hunk 2]
```diff
@@ -1,9 +1,12 @@
 class CharacterizeJob < ActiveFedoraIdBasedJob
   queue_as :characterize
 
+  # Calls characterization, saves the generic_file, and queues a job to create derivatives.
+  # Sets original_name property to be same as that of original_file thus over riding characterization.
   def perform(id)
     @id = id
-    CurationConcerns::CharacterizationService.run(generic_file)
```
Comment (jcoyne) [inline curation_concerns-models/app/jobs/characterize_job.rb]: This change is at cross purposes with #347 What's the best way for us to resolve that?

File: Gemfile (hunk 1 above)
Comment (grosscol) [inline Gemfile]: Yes, that's part of the UMICH CONFIG remove this when you squash commit.

File: curation_concerns-models/app/jobs/characterize_job.rb (hunk 2 above)
Comment (grosscol) [inline curation_concerns-models/app/jobs/characterize_job.rb]: I'll refactor it.  I'll have to do it anyway once the characterization is done in hydra works.

**Pull request closed without merging**